src/track/beatmap.cpp

```diff
@@ -490,36 +490,6 @@ void BeatMap::removeBeat(double dBeatSample) {
     emit updated();
 }
 
-void BeatMap::moveBeat(double dBeatSample, double dNewBeatSample) {
```

Comment:
Are these changes to BeatMap intended?  This feels like something left over from a merge.